assets/webpack.config.js

```diff
@@ -0,0 +1,105 @@
+const path = require('path');
```

Comment:
[For reference, here's how Phoenix does it upstream](https://github.com/phoenixframework/phoenix/blob/v1.5.3/installer/templates/phx_assets/webpack.config.js)